packages/server/tests/acceptance/rpc.spec.ts

```diff
@@ -476,6 +488,36 @@ describe('RPC Server Acceptance Tests', function() {
         });
     });
 
+    describe('eth_getCode', () => {
+        it('should execute "eth_getCode" for contract evm_address', async function () {
+            const evmAddress = basicContractReceipt.contractId.toSolidityAddress();
+            const res = await relay.call('eth_getCode', [evmAddress]);
+            expect(res).to.eq(basicContractJson.deployedBytecode);
+        });
+
+        it('should execute "eth_getCode" for contract with id converted to evm_address', async function () {
+            const evmAddress = Utils.idToEvmAddress(basicContractReceipt.contractId.toString());
+            const res = await relay.call('eth_getCode', [evmAddress]);
+            expect(res).to.eq(basicContractJson.deployedBytecode);
+        });
+
+        it('should fail "eth_getCode" for non-existing contract', async function () {
+            const res = await relay.call('eth_getCode', [NON_EXISTING_ADDRESS]);
+            expect(res).to.eq('0x0');
+        });
+
+        it('should fail "eth_getCode" for account evm_address', async function () {
+            const evmAddress = Utils.idToEvmAddress(accounts[2].accountId.toString());
+            const res = await relay.call('eth_getCode', [evmAddress]);
+            expect(res).to.eq('0x0');
+        });
+
+        it('should fail "eth_getCode" for account alias', async function () {
+            const res = await relay.call('eth_getCode', [mirrorPrimaryAccount.alias]);
```

Comment:
in order to be consistent, lets use the same `accounts[2]` but do `Utils.idToEvmAddress(accounts[2].alias)`